server/backend/api/app/main.py

```diff
@@ -104,11 +104,11 @@ def main():
                    tags=["FAM User Role Assignment"])
 app.include_router(router_forest_client.router,
                    prefix=apiPrefix + '/forest_clients',
-                   dependencies=[Depends(jwt_validation.authorize)],
+                   dependencies=[Depends(router_guards.authorize)],
```

Comment:
user need to be app admin or delegated admin of at least one application in order to call search forest client, and search idir, bceid user apis